source/Calamari.Shared/Integration/Processes/CommandLineRunner.cs

```diff
@@ -50,6 +56,22 @@ public CommandResult Execute(CommandLineInvocation invocation)
             }
         }
 
+        protected virtual List<ICommandOutput> GetCommandOutputs(CommandLineInvocation invocation)
+        {
+            var outputs = new List<ICommandOutput>
+            {
+                new ServiceMessageCommandOutput(variables)
```

Comment:
Only two or so callers didn't pass `ServiceMessageCommandOutput` and it shouldn't affect them if it was included, so it's included as standard.